.gitignore

```diff
@@ -1,3 +1,4 @@
 /.cask/
 /electric-operator.elc
 /.ecukes-failing-scenarios
+*~
```

Comment:
Not important, but FYI you can put things like this in a [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) so that you don't need to add this line to every repo that you work on.

Reply:
Thanks for this advice, I've now implemented that.